packages/dd-trace/src/ci-visibility/exporters/agentless/writer.js

```diff
@@ -14,7 +20,22 @@ class Writer extends BaseWriter {
   }
 
   _sendPayload (data, _, done) {
-    makeRequest(data, this._url, (err, res) => {
+    const options = {
+      path: '/api/v2/citestcycle',
+      method: 'POST',
+      headers: {
+        'dd-api-key': process.env.DATADOG_API_KEY || process.env.DD_API_KEY,
```

Comment:
What happens if neither env var is defined?

Reply:
agentless is not activated: https://github.com/DataDog/dd-trace-js/blob/master/ci/init.js#L18, so we wouldn't reach this point